### PR TITLE
Changing color values using  if statement.

### PR DESCRIPTION
**Pull request by christiangubana**
This PR fixes a small check on a color value passed as a param in the URL.

##How to test this PR

- checkout to this branch
- type npm install 
- cd into ```express-based``` folder
- run the file as follow from terminal
  a. ``node server.js``
  b. ``curl http://localhost:3000/color/:name``

### CODE REVIEW

File: express-basic/server.js [hunk 1]
```diff
@@ -22,6 +22,10 @@ app.use('/color/:name', function(req, res) {
   
   if('red' === name) {
     code = '#FF0000'
+  } else if('green' === name) {
+    code = '#00ff00'
+  } else if ('blue' === name) {
```
Comment (rjrodger) [inline express-basic/server.js]: consistent formatting - always use `if(...)` rather than `if ()`

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: express-basic/server.js [hunk 2]
```diff
@@ -22,6 +22,10 @@ app.use('/color/:name', function(req, res) {
   
   if('red' === name) {
     code = '#FF0000'
```
Comment (rjrodger) [inline express-basic/server.js]: let's use lower case for all the colors as per your changes

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: express-basic/server.js (hunk 1 above)
Comment (christiangubana) [reply express-basic/server.js]: Alright, thanks, I'll get this changed shortly.

File: express-basic/server.js (hunk 2 above)
Comment (christiangubana) [reply express-basic/server.js]: Ok, this is updated.

**Pull request merged**